Automattic/jetpack
# My plan: update free plan page

Fixes #10995 

#### Changes proposed in this Pull Request:

* Update the Free plan page on My Plan:
  * Make plan feature cards more useful and actionable.
  * Reorder cards so that Security & Performance are the top two focuses.
  * Remove Premium plan promo.

#### Testing instructions:

* Go to your Jetpack site or [create a new one](https://jurassic.ninja/create/).
* Connect Jetpack if it isn't already.
* Go back to wp-admin and open the `My Plan` page.
* Make sure the page is cleaner from distractions, and that all cards have a corresponding action.

#### Before

![image](https://user-images.githubusercontent.com/390760/53416915-5c876700-39cc-11e9-806f-747320a529eb.png)


#### After


![image](https://user-images.githubusercontent.com/390760/53813717-332f8380-3f56-11e9-8c25-f38b42d7534b.png)


#### Proposed changelog entry for your changes:

* My Plan page is now more useful and actionable for free plans.
